build-pipelines-PR.yml

```diff
@@ -225,224 +151,6 @@ jobs:
       zipAfterPublish: false
       modifyOutputPath: false
       selectOrConfig: config
-  - task: PowerShell@2
-    displayName: Download kubectl
-    continueOnError: True
-    enabled: True
-    inputs:
-      targetType: inline
-      script: >
-        New-Item -Path './src/dsc/bin/${{ parameters.BuildConfiguration }}/net6.0/win-x64/publish/kubectl' -ItemType Directory
-
-        New-Item -Path './src/dsc/bin/${{ parameters.BuildConfiguration }}/net6.0/win-x64/publish/kubectl/win' -ItemType Directory
-
-        curl https://dl.k8s.io/release/v1.21.2/bin/windows/amd64/kubectl.exe -OutFile ./src/dsc/bin/${{ parameters.BuildConfiguration }}/net6.0/win-x64/publish/kubectl/win/kubectl.exe
-
-
-        New-Item -Path './src/dsc/bin/${{ parameters.BuildConfiguration }}/net6.0/osx-x64/publish/kubectl' -ItemType Directory
-
-        New-Item -Path './src/dsc/bin/${{ parameters.BuildConfiguration }}/net6.0/osx-x64/publish/kubectl/osx' -ItemType Directory
-
-        curl https://dl.k8s.io/release/v1.21.2/bin/darwin/amd64/kubectl  -OutFile ./src/dsc/bin/${{ parameters.BuildConfiguration }}/net6.0/osx-x64/publish/kubectl/osx/kubectl
-
-
-        New-Item -Path './src/dsc/bin/${{ parameters.BuildConfiguration }}/net6.0/linux-x64/publish/kubectl' -ItemType Directory
-
-        New-Item -Path './src/dsc/bin/${{ parameters.BuildConfiguration }}/net6.0/linux-x64/publish/kubectl/linux' -ItemType Directory
-
-        curl https://dl.k8s.io/release/v1.21.2/bin/linux/amd64/kubectl  -OutFile ./src/dsc/bin/${{ parameters.BuildConfiguration }}/net6.0/linux-x64/publish/kubectl/linux/kubectl
-
-  - task: CopyFiles@2
-    displayName: Collect files for .zip (Windows)
```

Comment:
Simpler is great, but for some of these tasks I wonder if it is worth keeping parity with release pipeline. So PR build and release pipeline stay as close as possible aside from tasks that cant be ran from fork. I am open either way, just thinking.

Reply:
This will reduce the time PR builds take and it is already validating if it restores, builds and UT's. I think these copy files, zip tasks wouldn't change much. if you think if we want to keep parity then we can't save much on the build time. Just a suggestion on pros/cons of doing it simple.

Reply:
sounds good. let go with removing them.